examples/k8s/README.md

```diff
@@ -0,0 +1,30 @@
+# Kubernetes executor
```

Comment:
This is the very beginning of a README for the k8s example. The commands below allowed me to get a working example on a mac.